code/game/gamemodes/game_mode.dm

```diff
@@ -185,6 +185,11 @@
 /datum/game_mode/proc/generate_station_goals(greenshift)
 	var/goal_budget = greenshift ? INFINITY : CONFIG_GET(number/station_goal_budget)
 	var/list/possible = subtypesof(/datum/station_goal)
+	if(!(SSmapping.empty_space))
+		for(var/datum/station_goal/goal in possible)
+			if(goal.requires_space == 1)
```

Comment:
is there a requires == 2?

Reply:
no, it's just 1 if it does require space, but if it bothers you I can just change it to TRUE/FALSE